### PR TITLE
added two serial benchmarks "hamming and soli" with their respe…

### PR DESCRIPTION
**Pull request by ernestmusong**
…ctive dune files with corresponding runs in run_config.json

### CODE REVIEW

File: benchmarks/hamming/dune [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+(executable
+  (name hamming)
+  (modes byte exe))
+  (alias (name hamming)
```
Comment (Sudha247) [inline benchmarks/hamming/dune:5]:
```suggestion
  (alias (name buildbench)
```
`buildbench` is the alias used for all sequential benchmarks

File: dependencies/packages/ocaml/ocaml.5.0.0+stable/opam [hunk 2]
```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
```
Comment (Sudha247) [inline dependencies/packages/ocaml/ocaml.5.0.0+stable/opam:1]: The compiler opam files need not be added to the PR, they are pulled from the `ocaml-version` specified during the run. I suppose the benchmarks in this PR don't require any updates in `dependencies/packages`

File: run_config.json [hunk 3]
```diff
@@ -1520,6 +1520,32 @@
           "params": ""
         }
       ]
+    },
+    {
+      "executable": "benchmarks/soli/soli.exe",
+      "name": "soli",
+      "tags": [
+        "1s_10s",
```
Comment (Sudha247) [inline run_config.json:1528]: This tag means that the running time of the benchmark is between 1s and 10s. Is that the case here?

**Review by Sudha247 — COMMENTED** [review]
Thanks for the PR @ernestmusong! The benchmark placement looks right, I have some suggestions below.

Are you able to report the times for the two benchmarks?

**Comment by Sudha247** [comment]
The dune file update looks right. You've removed the file [dependencies/packages/ocaml/ocaml.5.0.0+stable/opam](https://github.com/ocaml-bench/sandmark/pull/320/commits/ffdcf177efb53f2c5db8eee8ea024030e4863dab#diff-610c2a89b60831a11d87bd1b6e69e71909db908a9aa9571cb4de42bff6932b30), could you also remove the other files added in `dependencies/packages`?

File: run_config.json (hunk 3 above)
Comment (ernestmusong) [reply run_config.json:1528]: yes

File: dependencies/packages/ocaml/ocaml.5.0.0+stable/opam (hunk 2 above)
Comment (ernestmusong) [reply dependencies/packages/ocaml/ocaml.5.0.0+stable/opam]: I have removed the opam file from "ocaml.5.0.0+stable"

**Comment by ernestmusong** [comment]
Hi @Sudha247  I've made all changes as required, please review.

**Comment by ernestmusong** [comment]
> The dune file update looks right. You've removed the file [dependencies/packages/ocaml/ocaml.5.0.0+stable/opam](https://github.com/ocaml-bench/sandmark/pull/320/commits/ffdcf177efb53f2c5db8eee8ea024030e4863dab#diff-610c2a89b60831a11d87bd1b6e69e71909db908a9aa9571cb4de42bff6932b30), could you also remove the other files added in `dependencies/packages`?

**Comment by Sudha247** [comment]
> Do you mean I should remove all the other opam files in all packages including packages not commited by me?

Yes. Although unintentional, I think they were added in this PR. Remove all the files in `dependencies/packages` in this PR.

**Comment by ernestmusong** [comment]
> > Do you mean I should remove all the other opam files in all packages including packages not commited by me?
> 
> Yes. Although unintentional, I think they were added in this PR. Remove all the files in `dependencies/packages` in this PR.

ok

**Comment by Sudha247** [comment]
Hi, please don't remove the files that were not added in the PR. You can see the files added in this PR at https://github.com/ocaml-bench/sandmark/pull/320/commits/38001356cdeaa1d2df2ab334b7e28cd065612162

**Pull request closed without merging**